src/sealVerseData.ts

```diff
@@ -0,0 +1,10 @@
+import Option from 'components/Dropdown/Option'
+
+export default [
+  { label: 'SealCred', value: 'https://sealcred.xyz' },
+  {
+    label: 'SealCred Echo',
+    value: 'https://echo.sealcred.xyz',
+  },
+  // { label: 'Sealcaster', value: 'https://sealcaster.xyz' },
```

Comment:
Should we uncomment this?

Reply:
I guess not for now. If I uncomment it, then the option with Sealcaster will be available